src/components/Searcher.js

```diff
@@ -982,8 +1020,7 @@ class Searcher extends React.Component<SearcherDefaultProps, SearcherProps, Sear
    * When a filter is removed, signal with weight 0 is created.
    */
   addFacetFilterSignal(facetFilter: FacetFilter, weight: number) {
-    const queryDocuments = this.state.response ? this.state.response.documents : null;
-    const querySignal = queryDocuments && queryDocuments.length >= 1 ? queryDocuments[0].signal : null;
+    const querySignal = this.getDefaultQuerySignal();
```

Comment:
Again here I'd add a comment saying what you're going to be putting in the docId field of the signal.

We will want to add this to the documentation on Answers, too... if you can create a ticket in Jira to do that that says something like "Document the values in Signals for Facet Filters, Promotions, and AutoComplete" that'd be great — it really belongs in the signals documentation more than the Search UI docs....

Reply:
Created https://jira.attivio.com/browse/PLAT-44324 for the documentation.